development/build/manifest.js

```diff
@@ -42,7 +39,6 @@ function createManifestTasks({
         );
         const result = mergeWith(
           cloneDeep(baseManifest),
-          process.env.BARAD_DUR ? cloneDeep(baradDurManifest) : {},
```

Comment:
@Gudahtt i'm not actually sure how to use the feature flag here to selectively include externally_connectable property values in the manifest. In fact I don't even think env vars specified in the builds.yml are available in this build context

Reply:
Yeah, this is not a feature of our build script. We could leave something hard-coded here for a specific feature, but probably easiest to make the necessary manifest change to the Flask manifest.

Reply:
Perhaps we could add a comment to `builds.yml` to call out this dependency? e.g. to say that the multichain-api feature requires this manifest change.

Or we could do something like... update this script to check features based on the build type, and apply a feature-specific manifest file.

Reply:
Replacing code fences with BARAD_DUR env var https://github.com/MetaMask/metamask-extension/pull/27782/commits/1230a38598322348d4f0c8d0625c645dff87057f

Renaming BARAD_DUR env var to MULTICHAIN_API  https://github.com/MetaMask/metamask-extension/pull/27782/commits/e381705ec177b2328df2253cc65546f0f8e584c4